portfolio/src/main/webapp/static/scripts/home.js

```diff
@@ -47,3 +47,60 @@ window.addEventListener('scroll', function(e) {
     ticking = true;
   }
 });
+
+
+const drawProgrammmingExperienceChart = () => {
```

Comment:
It doesn't really matter, but putting the chart functions into a different file might be good bc those are mainly hard coded data.